web-api/migrations/20230216164200_document.sql

```diff
@@ -0,0 +1,21 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+ALTER TABLE document
+    -- remove default snippet when the feature flag ET-3837 is removed
```

Comment:
We will need another migration file to remove them so I don't think this comment is relevant

Reply:
i'll remove the comment, but we should keep that in mind.